src/rdkafka.c

```diff
@@ -1048,20 +1048,48 @@ static void rd_kafka_stats_emit_all (rd_kafka_t *rk) {
 			   " \"max\":%"PRId64","
 			   " \"avg\":%"PRId64","
 			   " \"sum\":%"PRId64","
+                           " \"stddev\": %"PRId64","
+                           " \"mean\": %"PRId64","
+                           " \"histoor\": %"PRId64","
```

Comment:
i think `historr` more logically belongs after `p99_99`. also maybe call it just `oor`? 
it's a bit funny to have both `avg` and `mean`, though (without looking at the hdr implementation), i'm guessing they will be slightly different due to approximations in the histogram code. Perhaps just have `avg`? also maybe put this immediately before `stddev`? also, if you do keep `mean`, I'd put that before `stddev`.

Reply:
`min, max, sum, mean, stddev, p50, p75, p90, p95, p99, p99_99, oor, cnt`
`min, max, sum, avg, stddev, p50, p75, p90, p95, p99, p99_99, oor, cnt`
`min, max, sum, avg, mean, stddev, p50, p75, p90, p95, p99, p99_99, oor, cnt`

i'd choose one of the above metric sets / orderings, i think probably erring on the side of the middle one.

Reply:
This is a JSON document so the order does not really matter:
> An object is an unordered set of name/value pairs

I'll remove `mean` for now. :+1:

Reply:
Renaming "histoor" -> "outofrange"